ssim/ssimlib.py

```diff
@@ -78,7 +78,11 @@ def ssim_value(self, img2):
         Returns:
```

Comment:
Could you update the name of the `img2` argument and the associated docstring to indicate that you can now pass an `SSIM` object instance?